dashboard/test/models/pd/workshop_test.rb

```diff
@@ -363,13 +363,13 @@ class Pd::WorkshopTest < ActiveSupport::TestCase
   end
 
   test 'plp' do
-    assert_nil @workshop.professional_learning_partner
+    assert_nil @workshop.regional_partner
 
     # Now create a plp associated with the organizer
```

Comment:
I plan to update minor references to the _concept_ of a professional learning partner like variable names and comments mostly as I go. It's still conceptually valid, so it doesn't seem completely necessary to change em just for the sake of changing em.